### ISSUE TITLE
Maintainer Status and future ownership?

### ISSUE THREAD

**Issue by PeteMichaud** [ArianaKhit/text-snippets-obsidian#43]
Hi @ArianaKhit! Before we start any discussion, we want to give you a massive thanks for all the time and effort you've put into creating this plugin. A ton of people, myself included, use the plugin for our Obsidian workflows. You've kept up the spirit of Obsidian by helping us do more inside our favorite PKM tool.

We understand that not everyone has the time to maintain and respond to issues. Active development can be challenging when there are other things to attend to and that sometimes it can be taxing to think of "pending work". You're probably well aware that there's a number of issues with some people eagerly awaiting fixes.

A few of us would love to step in and help out in whatever way we can. Although some people have offered to implement fixes, it's hard to merge them without the right permissions. On that note, we wanted to ask if you were willing to transfer ownership of the repo to the github community hub organization maintained by the Obsidian community. It's a difficult decision to make right away so please take your time and let us know how you feel about this. In case you wish that we take some other route like forking the repo, that's fine by us as well.

At the end of the day, we're happy that you've done so much for the community and we want to take your work further. We hope you can let us know soon! In the event that it takes longer than a month for you to respond, we'll be continuing with bug fixes in a fork and we can decide where to go from there.

**Comment by ArianaKhit** [comment]
Hi @PeteMichaud! Thank you for your help! 
I am ready to open access to the repository to those who want to participate in improving the project, so you can merge fixes.

**Comment by ArianaKhit** [comment]
> it had the option to define snippets in a folder

Hi! That sounds good, and I think it can be useful. But the problem in multiline snippets is not in ```$nl$```, it is needed because snippets are separated from each other using ```\n```, so to avoid troubles with this character it is necessary to use some rules in snippets creation

**Comment by TinyAtoms** [comment]
Yeah, i guessed that was the reason. Separating multiline snippets to files
in a folder gets around that issue, right? Since those won't be separated
by a newline, we wouldn't need a special character. Anyway, if i added that
functionality, would you consider merging the pull request? (I am not
particularly experienced with typescript or the obsidian API, but I'm
pretty sure i can make something that works)

**Comment by ArianaKhit** [comment]
> would you consider merging the pull request?

Sure

**Comment by mayurankv** [comment]
I feel like this thread may still be relevant
